bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -254,7 +254,11 @@ <h3 id="os-operators-headline" class="tween">
               <div>
                 {% if l10n_has_tag('mwc_2015') %}
```

Comment:
Since you're touching this file, we should also take the occasion to get rid of this tag, all active locales have the strings translated.